portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -12,6 +12,36 @@ definition {
 		Button.clickSave();
 	}
 
+	macro addLanguageForOBJ {
+		Click(locator1 = "Button#PLUS");
+
+		Type(
+			key_label = ${key_label},
+			locator1 = "DataSet#SPECIFIC_INPUT",
+			value1 = ${key_text1});
+
+		PortletEntry.save();
+
+		Refresh();
+
+		Click(locator1 = "DataSet#SELECT_LOCALIZATION_BUTTON");
+
+		SelectFrame(locator1 = "DataSet#SELECT_FRAME_LOCAL");
+
+		Click.javaScriptClick(
+			key_menuItem = ${key_menuItem},
+			locator1 = "MenuItem#MENU_ITEM_SPECIFIC");
+
+		Type(
+			key_label = ${key_type},
+			locator1 = "DataSet#SPECIFIC_INPUT",
+			value1 = ${key_text2});
+
+		Click(locator1 = "DataSet#SELECT_FRAME_DEFOCUS");
+
+		PortletEntry.save();
+	}
+
```

Comment:
This macro now has the proper name but it is placed in the wrong file. This macro is related to objects so, instead of putting it in the DataSetAdmin.macro it should be placed in the ObjectAdmin.macro file. Please relocate this.